test/e2e/run_tests.sh

```diff
@@ -93,26 +128,35 @@ for POLICY_DIR in "$TESTS_ROOT_DIR"/*; do
             export cri_resmgr_cfg=$CFG_FILE
         done
         export_var_files "$POLICY_DIR"
+        export_vm_files "$POLICY_DIR/vm-files"
         for TOPOLOGY_DIR in "$POLICY_DIR"/*; do
             if ! [ -d "$TOPOLOGY_DIR" ]; then
                 continue
             fi
             if ! [[ "$(basename "$TOPOLOGY_DIR")" =~ .*"$TESTS_TOPOLOGY_FILTER".* ]]; then
                 continue
             fi
+            if [ "$(basename "$TOPOLOGY_DIR")" == "vm-files" ]; then
+                continue
+            fi
             (
                 export_var_files "$TOPOLOGY_DIR"
+                export_vm_files "$TOPOLOGY_DIR/vm-files"
                 vm="$(basename "$TOPOLOGY_DIR")"
                 export vm
                 for TEST_DIR in "$TOPOLOGY_DIR"/*; do
-                    if ! [ -d "$TEST_DIR" ]; then
+                    if ! [ -d "$TEST_DIR" ] ; then
```

Comment:
nitpick: extra space slipped through/in

Reply:
uh-oh. need to improve "git diff" self-review before pushing...
Thanks!